js/script-dfi.js

```diff
@@ -91,6 +91,9 @@ jQuery( document ).ready( function ( $ ) {
                     text: DFI_SPECIFIC.mediaSelector_buttonText
                 },
                 multiple: false,
+                library: {
+                    type: [ 'image', 'video' ]
```

Comment:
Should be `image` only, the plugin doesn't support `video` yet.

Reply:
Alright, updated PR.